README.md

```diff
@@ -1,2 +1,64 @@
 # Portfolio
-This is my first Portfolio
+
+
+# Project Name
+
+> Portfolio
+
+
+## Built With
+
+- HTMl
+- CSS
+
+
+## Live Demo (if available)
+
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+git clone https://github.com/victyr/Portfolio.git
+
+index.html
+styles.css
+
+### Prerequisites
+
+### Setup
+
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Walugembe Victor**
+
+- GitHub: [@githubhandle](https://github.com/victyr)
+- 
+
```

Comment:
[OPTIONAL] I recommend you change the generic `@githubhandle` handle and change it to your GitHub handle, it'll make your documentation look more professional.